Makefile

```
@@ -80,6 +80,14 @@ tidy: ## Tidy go modules
 run: build ## Build and run
 	./bin/$(BINARY_NAME)
 
+dev-sync: build install ## Build + install + sync templates -> this dev project (.claude/, .moai/)
```

checkmake (0.3.2)

[warning] Target body for "dev-sync" exceeds allowed length of 5 lines (6).

(maxbodylength)